components/sections/blog/stories/blog.stories.tsx

```diff
@@ -0,0 +1,1128 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import BlogComponent from "../index";
+import { Sections, SectionsProps, Variants } from "types";
+import {
+  blogSchema,
+  blogInitialValue,
+} from "@webriq-pagebuilder/sanity-plugin-schema-default";
+import { filterArgsByVariant } from "components/common";
+
+const args: Variants = {
```

Comment:
I think for default component arguments this long maybe we could define it in a common file instead (if they're not exported from the schema packages atm). What do you think?